cmd/clusterctl/pkg/client/cluster/cert_manager.go

```diff
@@ -110,10 +115,11 @@ func (cm *certMangerClient) EnsureWebHook() error {
 	}
 
 	// Waits for for the cert-manager WebHook to be available.
```

Comment:
not a blocker...but since we have the `log.Info` message I don't think we need this comment as well.